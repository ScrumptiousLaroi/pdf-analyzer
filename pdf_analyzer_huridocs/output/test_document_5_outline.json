{
    "title": "API Integration Guide",
    "outline": [
        {
            "level": "H2",
            "text": "Getting Started",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Prerequisites",
            "page": 1
        },
        {
            "level": "H2",
            "text": "System Requirements",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Installation Steps",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Authentication",
            "page": 1
        },
        {
            "level": "H2",
            "text": "API Endpoints",
            "page": 1
        },
        {
            "level": "H2",
            "text": "User Management",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Creating Users",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Updating User Information",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Data Operations",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Error Handling",
            "page": 2
        }
    ]
}