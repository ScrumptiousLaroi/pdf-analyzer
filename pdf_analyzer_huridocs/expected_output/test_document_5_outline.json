{
    "title": "API Integration Guide",
    "outline": [
        {
            "level": "H1",
            "text": "Getting Started",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Prerequisites",
            "page": 1
        },
        {
            "level": "H3",
            "text": "System Requirements",
            "page": 1
        },
        {
            "level": "H3",
            "text": "Installation Steps",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Authentication",
            "page": 1
        },
        {
            "level": "H1",
            "text": "API Endpoints",
            "page": 1
        },
        {
            "level": "H2",
            "text": "User Management",
            "page": 1
        },
        {
            "level": "H3",
            "text": "Creating Users",
            "page": 1
        },
        {
            "level": "H3",
            "text": "Updating User Information",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Data Operations",
            "page": 1
        },
        {
            "level": "H1",
            "text": "Error Handling",
            "page": 1
        }
    ]
}